StudyPathTraker/StudyPathTraker/Navigation/SPLightNavigationControllerDelegate.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  SPLightNavigationControllerDelegate.swift
+//  StudyPathTraker
+//
+//  Created by Rafael Lopez on 5/30/18.
+//  Copyright © 2018 Jerti. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+protocol backItem {
```

Comment:
I think this is unnecessary you could add this same method into a UIViewController extension.

Reply:
since there is no other elements that will use this protocol. Always that you think in a protocol, first think how many objects would use that and whether is necessary make a mock. If the answer for this is just one and the other is no necessary, means that you don't need a protocol.

Reply:
ready, removed